client/data/payment-activity/hooks.ts

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../constants';
+import { PaymentActivityState } from './types';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const usePaymentActivityData = ( query: any ): PaymentActivityState =>
```

Comment:
💭 Similar to the comment above, can we define the types that we require `query` to contain?

```suggestion
// Something like...
interface PaymentActivityQuery {
	date_start: string;
	date_end: string;
	timezone?: string;
}
export const usePaymentActivityData = ( query: PaymentActivityQuery ): PaymentActivityState =>
```

Reply:
Thanks for the suggestion made changes: https://github.com/Automattic/woocommerce-payments/pull/8589/commits/2a0d4ebea23fba0d19690da3d86b0c3919a7317e